Services.Tests/SearchServiceTest.cs

```diff
@@ -0,0 +1,343 @@
+using Models;
+using Moq;
+using NUnit.Framework;
+using Repositories;
+using Repositories.Tests.DataSources;
+using Services.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace Services.Tests
+{
+    /// <summary>
+    /// SearchServiceTest
+    /// </summary>
+    /// <seealso cref="IProjectRepository" />
+    [TestFixture]
+    public class SearchServiceTest
+    {
+        protected ISearchService Service;
+        protected Mock<IProjectRepository> RepositoryMock;
+
+        /// <summary>
+        /// Initialize runs before every test
+        /// Mock the repository
+        /// Initialize the service with reflection
+        /// </summary>
+        [SetUp]
+        public void Initialize()
+        {
+            // Mock the repository
+            RepositoryMock = new Mock<IProjectRepository>();
+
+            // Create the service with reflection
+            Type serviceType = typeof(SearchService);
```

Comment:
You don't have to use reflection to create the object. You can create the service with new SearchService();
The base unittest class uses reflection due to the fact that it doesn't know which service it has to create. With reflection you are able to use generics. You don't need generics here so reflection is not necessary.

Reply:
I know, the reason that i did it with reflection is that it keeps the unit tests uniform.